packages/parchment/lib/src/convert/markdown.dart

```diff
@@ -210,8 +495,18 @@ class _ParchmentMarkdownEncoder extends Converter<Delta, String> {
     } else {
       if (close) return; // no close tag needed for simple blocks.
 
-      final tag = kSimpleBlocks[block];
+      final tag = simpleBlocks[block];
       buffer.write(tag);
     }
   }
 }
+
+extension<E> on Iterable<E> {
+  Iterable<E> reverse() {
```

Comment:
I've tested this functions performance (I suspected it's not going to perform well) and I found that it's way faster to do `iterable.toList().reversed` than to do `iterable.reverse()`. Actually it's 1350 times faster!